lib/content-services/src/lib/breadcrumb/breadcrumb.component.spec.ts

```diff
@@ -300,4 +300,26 @@ describe('Breadcrumb', () => {
         expect(component.route[3].id).toBe('test-id');
         expect(component.route[3].name).toBe('test-name');
     });
+
+    it('should set title based on selectedRowItemsCount', () => {
+        component.transform = (transformNode) => {
+            transformNode.id = 'test-id';
+            transformNode.name = 'test-name';
+            return transformNode;
+        };
+        (component.folderNode as any) = {
```

Comment:
It can be just: 
component.folderNode = {
  path: { elements: [{ id: 'element-1-id', name: 'element-1-name' }] }
} as Node; 

Will be better than using any.